streams/src/main/java/org/apache/kafka/streams/processor/internals/StateDirectory.java

```diff
@@ -681,4 +698,24 @@ public int hashCode() {
         }
     }
 
+    public static class BackoffRecord {
+        private long attempts;
+        private long lastAttemptMs;
+        private static final ExponentialBackoff EXPONENTIAL_BACKOFF = new ExponentialBackoff(1, 2, 10000, 0.5);
```

Comment:
should we change it to "exponentialBackoff" to keep it uniform as previous instances.
https://github.com/apache/kafka/blob/a9f3ea2dd34f8704f880bf58af1ff3a3f1070fbb/clients/src/main/java/org/apache/kafka/clients/consumer/internals/RequestState.java#L30C40-L30C58

Reply:
Naming static variables is different.